charts/rekor/templates/server/createtree-job.yaml

```diff
@@ -14,7 +14,11 @@ spec:
       containers:
         - name: {{ template "rekor.createtree.fullname" . }}
           image: "{{ template "rekor.image" .Values.createtree.image }}"
-          args: ["--namespace=$(NAMESPACE)", "--configmap={{ template "rekor.config" . }}", "--display_name=rekortree","--admin_server={{ .Values.trillian.logServer.name }}.{{ .Values.trillian.forceNamespace }}:{{ .Values.trillian.logServer.portRPC}}"]
+          args: ["--namespace=$(NAMESPACE)",
+            "--configmap={{ template "rekor.config" . }}",
+            "--display_name=rekortree","--admin_server={{ .Values.trillian.logServer.name }}.{{ .Values.trillian.forceNamespace }}:{{ .Values.trillian.logServer.portRPC}}",
```

Comment:
nit: please move each flag to a separate line for easier reading.

Reply:
done